pkg/handlers/authorization/handler.go

```diff
@@ -83,9 +73,8 @@ func sanitizeHeaders(req *http.Request) {
 
 func getBearerTokenFrom(req *http.Request) string {
 	parts := strings.SplitN(req.Header.Get(headerAuthorization), " ", 2)
-	if len(parts) > 0 && parts[0] == "Bearer" {
+	if len(parts) > 1 && parts[0] == "Bearer" {
 		return parts[1]
 	}
-	log.Trace("No bearer token found on request. Returning ''")
```

Comment:
Seems this would be useful to keep

Reply:
The same log message is present in the caller